docs/source/spark.rst

```diff
@@ -6,53 +6,48 @@ PySpark_.  Spark is growing to become a dominant name today in Big Data
 analysis alongside Hadoop, for which MRJob_ is possibly the dominant
 Python layer.
 
-Dask has several elements that appear to intersect this space and as such we
-often receive the following question:
+Dask has several elements that appear to intersect this space and as such we are often asked, 
+"How does Dask compare with Spark?"
 
-*How does Dask compare with Spark?*
+Summary
+-------
 
-Answering such comparison questions in an unbiased and informed way is hard,
-particularly when the differences can be somewhat technical.  This document
-tries to do this; we welcome any corrections.
+* You should probably use Spark if you have petabytes of JSON files, a simple 
+  workflow, and a thousand node cluster. 
+* You should probably use dask if you have 10s-1000s of gigabytes of binary or 
+  numeric data, complex algorithms, and a large   multi-core workstation.
+* If you have a terabyte or less of CSV or JSON data, forget both Spark *and* 
+  Dask and use Postgres_ or MongoDB_ instead.
 
-Brief Answer
+General
 ------------
 
-Apache Spark is more mature and better integrates with HDFS.  It handles
-resiliency and was originally built to scale up to thousands of workers.
-
-Dask is pip installable, doesn't use the Java Virtual Machine (JVM), and was
+Dask can be installed with conda or pip, doesn't use the Java Virtual Machine (JVM), and was
 originally built to handle numeric workloads in a large single workstation very
 efficiently.
 
+Apache Spark is more mature and better integrates with HDFS.  It handles
+resiliency and was originally built to scale up to thousands of workers.
 
-User-Facing Differences
+User-facing differences
 -----------------------
 
 Scale
 ~~~~~
 
-Spark began its life aimed at the thousand node cluster case.  As
-such it thinks well about worker failures and integration with data-local
-file systems like the Hadoop FileSystem (HDFS).  That being said, Spark can
-run in standalone mode on a single machine.
-
 Dask began its life building out parallel algorithms for numerical array
 computations on a single computer.  As such it thinks well about low-latency
 scheduling, low memory footprints, shared memory, and efficient use of local
-disk.  That being said dask can run on a :doc:`distributed cluster <distributed>`.
+disk.  That being said dask can run on a distributed cluster.
 
+Spark began its life aimed at the thousand node cluster case.  As
+such it thinks well about worker failures and integration with data-local
+file systems like the Hadoop FileSystem (HDFS).  That being said, Spark can
+run in standalone mode on a single machine.
```

Comment:
Same here.  Why was the order reversed?

Putting dask first makes it look like we're biased.  I have tried very very hard in this document to be impartial or to politely put Spark in a preferred position.  I get very nervous whenever this stance is reversed.  

This section is sufficiently delicate that I would appreciate it if we could minimize changes here, no matter how subtle.